platform/pipelines/steps-platform-build-api-tests.yaml

```diff
@@ -0,0 +1,45 @@
+steps:
+  - task: NuGetAuthenticate@1
+  - bash: |
+      echo "##vso[task.setvariable variable=APITESTS_CACHE_RESTORED]false"
+      echo "##vso[task.setvariable variable=APITESTS_NUGET_CACHE]$(Pipeline.Workspace)/api-tests/.nuget"
+
+  - task: Cache@2
+    displayName: 'Cache NuGet packages'
+    inputs:
+      key: 'nuget | "$(Agent.OS)-api-tests" | platform/tests/EducationBenchmarking.Platform.ApiTests/packages.lock.json,!**/bin/**,!**/obj/**'
+      restoreKeys: |
+        nuget | "$(Agent.OS)-api-tests"
+      path: $(APITESTS_NUGET_CACHE)
+      cacheHitVar: APITESTS_CACHE_RESTORED
+
+  - task: DotNetCoreCLI@2
+    displayName: 'Restore nuget packages'
+    condition: ne(variables.APITESTS_CACHE_RESTORED, 'true')
+    inputs:
+      command: restore
+      projects: 'platform/tests/EducationBenchmarking.Platform.ApiTests/*EducationBenchmarking.Platform.ApiTests.csproj'
```

Comment:
Is that an intentional asterisk here and below?

Reply:
Nope, not intentional - good spot.

Reply:
Push change